ui/src/app/settings/selectors/config.js

```diff
@@ -98,4 +98,26 @@ config.diskEraseWithQuick = createSelector(
   configs => getValueFromName(configs, "disk_erase_with_quick_erase")
 );
 
+/**
+ * Returns the MAAS config for MAAS name.
+ * @param {Object} - The redux state.
+ * @returns {String} Then MAAS name.
+ */
+config.maasName = createSelector(
+  [config.all],
+  configs => {
```

Comment:
With ES6, you can just say (as you do below):

```
configs => getValueFromName(configs, "maas_name");
```

Reply:
ah yes, I think I had a longer version initially and prettier changed it, I forgot to change it back